core/base-service/base.spec.js

```diff
@@ -321,63 +327,24 @@ describe('BaseService', function () {
     })
   })
 
-  describe('ScoutCamp integration', function () {
-    // TODO Strangly, without the useless escape the regexes do not match in Node 12.
-    // eslint-disable-next-line no-useless-escape
-    const expectedRouteRegex = /^\/foo(?:\/([^\/#\?]+?))(|\.svg|\.json)$/
-
-    let mockCamp
-    let mockHandleRequest
-
-    beforeEach(function () {
-      mockCamp = {
-        route: sinon.spy(),
-      }
-      mockHandleRequest = sinon.spy()
-      DummyService.register(
-        { camp: mockCamp, handleRequest: mockHandleRequest },
-        defaultConfig
-      )
+  describe('Express integration', function () {
+    let harness
+    beforeEach(async function () {
+      harness = new ExpressTestHarness()
```

Comment:
Have added this helper class to reduce boilerplate in setting up, starting, and issuing requests to, an empty express app.